purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -352,6 +354,7 @@ fun Purchases.getSubscriptionSkusWith(
 /**
  * Gets the StoreProduct for the given list of non-subscription skus.
  * @param [skus] List of skus
+ * @param [onError] Will be called after the purchase has completed with error
```

Comment:
nit (sorry 🙈 ) but technically the purchase might not have completed.. so maybe more like:
`Will be called if there was an error with the purchase` or something?

Reply:
Ahhh, good call! Updating the rest to match this 😊